authors/apps/articles/urls.py

```diff
@@ -1,11 +1,17 @@
 from django.urls import path
 
-from .views import ArticleAPIView, ArticleAPIDetailsView, RateArticleView
+from .views import (ArticleAPIView, ArticleAPIDetailsView, 
+            RateArticleView, LikeArticleView, LikeAPIDetailsView)
 
 app_name = "articles"
 
 urlpatterns = [
     path("articles/", ArticleAPIView.as_view()),
-    path('articles/<str:slug>/', ArticleAPIDetailsView.as_view(),name='retrieveUpdateDelete'),
-    path('articles/<str:slug>/rating/', RateArticleView.as_view())
-]
\ No newline at end of file
+    path('articles/<str:slug>/', ArticleAPIDetailsView.as_view(),
+         name='retrieveUpdateDelete'),
+    path('articles/<str:slug>/rating/', RateArticleView.as_view()),
+    path('articles/<str:slug>/liking/',
```

Comment:
@zellymiriam I think **like** is a better tense than **liking**

Reply:
Thanks, @PatrickCmd let me modify this.